libr/core/cconfig.c

```diff
@@ -2290,29 +2290,30 @@ static char *getViewerPath() {
 	return NULL;
 }
 
-R_API char* r_core_graph_cmd(RCore *core, char *r2_cmd) {
+R_API char* r_core_graph_cmd(RCore *core, char *r2_cmd, char *save_path) {
```

Comment:
Actually you are right... It was here because it was used to initialize the graph.cmd variable, but now it has no more sense to keep it here, I'll move it